configuration/defaults.go

```diff
@@ -6,4 +6,36 @@ const (
 	defaultDBPassword = "mysecretpassword"
 
 	defaultLogLevel = "info"
+
+	// DevModeRsaPrivateKey for signing JWT Tokens in Dev Mode
+	// ssh-keygen -f alm_rsa
+	DevModeRsaPrivateKey = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Just reuse what we already have in https://github.com/fabric8-services/fabric8-common/blob/master/configuration/configuration.go#L41

Reply:
ah yes! thanks!